### PR TITLE
1527 html example Table

### PR DESCRIPTION
**Pull request by ArKhmeln**

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableTestsExample.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4807586&bid=15993805)

**Comment by mf-code** [comment]
[4443] - https://transfer.sh/ajMqe/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4443] - https://transfer.sh/Zz8FO/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4443] - https://transfer.sh/10AEl6/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4443] - Allure report on Netlify: https://5e1c3fcf7f5254a57e5b0eb6--musing-swartz-6fc060.netlify.com

File: jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableTestsExample.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                .rowThat(containsValue("Colby Young", inColumn("Name")))
+                .all().rows(containsValue("0", inColumn("Phone")))
+                .no().rows(containsValue("Australopithecus", inColumn("Name")))
+                .atLeast(2).rows(containsValue("Sean", inColumn("Name")))
+                .exact(15).rows(containsValue("0800 1111", inColumn("Phone")));
+    }
+
+    @Test
+    public void tableParamTest() {
+        usersSetup.is().size(4);
```
Comment (dexguitar) [inline jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableTestsExample.java]: The test fails.
Did you mean usersSetup.is().size(**400**)?

File: jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableTestsExample.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+public class TableTestsExample implements TestsInit {
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        tablePagePerformance.shouldBeOpened();
+    }
+
+    @Test
```
Comment (dexguitar) [inline jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableTestsExample.java]: This whole test doesn't work.

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableTestsExample.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4807586&bid=15993805)

File: jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableTestsExample.java (hunk 1 above)
Comment (ArKhmeln) [reply jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableTestsExample.java]: fixed

File: jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableTestsExample.java (hunk 2 above)
Comment (ArKhmeln) [reply jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableTestsExample.java]: fixed

**Comment by mf-code** [comment]
[4450] - https://transfer.sh/mmkYk/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4450] - https://transfer.sh/VWReQ/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4450] - https://transfer.sh/BjM4j/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4450] - Allure report on Netlify: https://5e1d7abac447aca26d3a7423--musing-swartz-6fc060.netlify.com

**Pull request merged**